config/default/system.site.yml

```diff
@@ -8,7 +8,7 @@ slogan: ''
 page:
   403: /node/1212
   404: /node/1212
-  front: /node/1211
+  front: /node/113
```

Comment:
Config ignore? I just checked prod and it's still `1211`

Reply:
I'll revert this. The next release with have the new version of Config Ignore, and we can stop this from being exported.